code/modules/surgery/bodyparts/species_parts/lizard_bodyparts.dm

```diff
@@ -8,7 +8,7 @@
 	icon = 'icons/mob/species/lizard/bodyparts.dmi'
 	uses_mutcolor = TRUE
 	limb_id = SPECIES_LIZARD
-	is_dimorphic = TRUE //any made enby humans, while halcy made them gendered, making them todays biggest loser
+	is_dimorphic = TRUE
```

Comment:
```suggestion
```

you don't need to set the value of a var to `TRUE` when it is already set to `TRUE` on the parent object